modules/core/src/main/java/org/apache/synapse/message/store/impl/jms/JmsStore.java

```diff
@@ -712,4 +723,17 @@ private String nameString() {
     public void setProducer(MessageProducer producer) {
         this.producer = producer;
     }
+
+    public String resolveQueueName(Object param) {
```

Comment:
I feel like this method is to resolve all the vault-look ups in the connection string. In that case, why did we name this as 'resolveQueueName'?